.drone.yml

```diff
@@ -137,6 +137,45 @@ steps:
   commands:
   - export PULL_REQUEST=$DRONE_PULL_REQUEST && ./.ci/releasenotes-gen.sh
 
+---
+kind: pipeline
+type: docker
+name: tweet-releasenotes
+
+volumes:
+  - name: m2-cache
+    temp: {}
+
+steps:
+  - name: restore-cache
+    image: maven:3.8.4-openjdk-11
+    failure: ignore
+    volumes:
+      - name: m2-cache
+        path: /.m2
+    commands:
+      - ./.ci/drone-io.sh restore-maven-cache
+
+- name: tweet-releasenotes-gen
```

Comment:
We need execution of job by tag creation and/or manually.

You made it by each commit, not good.

Reply:
Changes Revert. LINK IN PR DESCRIPTION